src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px ;
+  width: $card-width;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    margin-bottom: 40px;
+    max-width: 100%;
+    min-height: 134px;
+    padding: 0 3px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $font-color-main;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+    align-self: flex-start;
+    font-weight: 400;
```

Comment:
It's the default value
remove in all cases
![image](https://user-images.githubusercontent.com/91411692/222675732-2c874b59-93fa-4eae-9add-f57c0ce9ebcb.png)

![image](https://user-images.githubusercontent.com/91411692/222675648-684def54-5cf1-4c04-b707-21d0503a2dfb.png)

```suggestion
```